apps/sms/views/shared/js/localization_helper.js

```diff
@@ -11,11 +11,13 @@
       'iframe.attachment-container'
     );
 
-    Array.forEach(attachmentContainers, function(attachmentContainer) {
-      var doc = attachmentContainer.contentDocument;
-      doc.documentElement.lang = navigator.mozL10n.language.code;
-      doc.documentElement.dir = navigator.mozL10n.language.direction;
-      navigator.mozL10n.translateFragment(doc.body);
+    return document.l10n.languages.then(([lang]) => {
```

Comment:
Yeah, I realized the same thing :)  I'll fix this in https://bugzilla.mozilla.org/show_bug.cgi?id=1207120 (probably renaming `.languages` in the process to `resolvedLanguages()` which returns a promise.